hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/utils/TestProtobufUtils.java

```diff
@@ -46,4 +49,40 @@ public void testUuidConversion() {
     UUID deserialized = fromProtobuf(protobuf);
     assertEquals(original, deserialized);
   }
+
+  @Test
+  public void testContainerCommandRequestProtoConversion() {
+    long containerID = 1L;
+    long localBlockID = 2L;
+    long bcsid = 3L;
+    String datanodeID = UUID.randomUUID().toString();
+    ContainerProtos.DatanodeBlockID.Builder blkIDBuilder =
+        ContainerProtos.DatanodeBlockID.newBuilder().setContainerID(containerID)
+            .setLocalID(localBlockID)
+            .setBlockCommitSequenceId(bcsid);
+    ContainerProtos.GetBlockRequestProto.Builder readBlockRequest =
+        ContainerProtos.GetBlockRequestProto.newBuilder().setBlockID(blkIDBuilder.build());
+
+    ContainerProtos.ContainerCommandRequestProto.Builder builder =
+        ContainerProtos.ContainerCommandRequestProto.newBuilder()
+            .setCmdType(ContainerProtos.Type.GetBlock)
+            .setContainerID(containerID)
+            .setDatanodeUuid(datanodeID)
+            .setGetBlock(readBlockRequest.build());
+
+    ContainerProtos.ContainerCommandRequestProto request = builder.build();
+    byte[] requestInBytes = request.toByteArray();
+
+    try {
+      request = ContainerProtos.ContainerCommandRequestProto.parseFrom(requestInBytes);
+      assertTrue(request.hasGetBlock());
+      assertEquals(ContainerProtos.Type.GetBlock, request.getCmdType());
+      assertEquals(containerID, request.getContainerID());
+      assertEquals(datanodeID, request.getDatanodeUuid());
+      assertEquals(localBlockID, request.getGetBlock().getBlockID().getLocalID());
+      assertEquals(bcsid, request.getGetBlock().getBlockID().getBlockCommitSequenceId());
+    } catch (InvalidProtocolBufferException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
I think you can just let it throw exception and exit. Change method signature accordingly too.
```suggestion
      request = ContainerProtos.ContainerCommandRequestProto.parseFrom(requestInBytes);
      assertTrue(request.hasGetBlock());
      assertEquals(ContainerProtos.Type.GetBlock, request.getCmdType());
      assertEquals(containerID, request.getContainerID());
      assertEquals(datanodeID, request.getDatanodeUuid());
      assertEquals(localBlockID, request.getGetBlock().getBlockID().getLocalID());
      assertEquals(bcsid, request.getGetBlock().getBlockID().getBlockCommitSequenceId());

```